ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx

```diff
@@ -213,11 +213,7 @@ export default function SingleSectionEnrollmentWorkflow (props: SingleSectionEnr
 
     const fileData =
     `${REQUIRED_ENROLLMENT_HEADERS.join(',')}
-    student,studentone
-    teacher,teacherone
-    ta,taone
-    observer,observerone
-    designer,designerone`
+     studentone, student`
```

Comment:
Thanks for finding (@melindakraft) and fixing (@pushyamig) my mistake!

See note in other comment about leading whitespace.

Reply:
Done